cms/static/js/views/utils/wiki_utils.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Provides WikiTranslation utilities for views to work with xblocks.
```

Comment:
can we move custom js files in meta_translation app ?

Reply:
I tried that but webpack gives errors which running paver update_assets.